manifests/install.pp

```diff
@@ -45,12 +45,17 @@
   # Examples of product tarballs from Atlassian
   # Core                - atlassian-jira-core-7.0.3.tar.gz
   # Software (pre-7)    - atlassian-jira-6.4.12.tar.gz
-  # Software (7 and up) - atlassian-jira-software-7.0.4-jira-7.0.4.tar.gz
+  # Software (7 to 7.1.8 ) - atlassian-jira-software-7.0.4-jira-7.0.4.tar.gz
+  # Software (7.1.9 and up) - atlassian-jira-software-7.1.9.tar.gz
 
-  if ((versioncmp($jira::version, '7.0.0') < 0) or ($jira::product_name == 'jira-core')) {
-    $file = "atlassian-${jira::product_name}-${jira::version}.${jira::format}"
+  if (versioncmp($jira::version, '7.1.9') < 0) {
+    if ((versioncmp($jira::version, '7.0.0') < 0) or ($jira::product_name == 'jira-core')) {
+      $file = "atlassian-${jira::product_name}-${jira::version}.${jira::format}"
```

Comment:
The new `else` case at the bottom looks identical to this one. Is this intentional? The comment above says
```
# Software (pre-7)    - atlassian-jira-6.4.12.tar.gz
```
vs.
```
# Software (7.1.9 and up) - atlassian-jira-software-7.1.9.tar.gz
```
If these are supposed to use the same format, the logic flow should probably reflect that.